### ISSUE TITLE
Many-Joint Robot Arm Control with Recurrent Spiking Neural Networks. (arXiv:2104.04064v1 [cs.RO])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#5757]
Many-Joint Robot Arm Control with Recurrent Spiking Neural Networks. (arXiv:2104.04064v1 [cs.RO])<br>
https://ift.tt/3d7iuLS<br>
<p>In the paper, we show how scalable, low-cost trunk-like robotic arms can be constructed using only basic 3D-printing equipment and simple electronics. The design is based on uniform, stackable joint modules with three degrees of freedom each. Moreover, we present an approach for controlling these robots with recurrent spiking neural networks. At first, a spiking forward model learns motor-pose correlations from movement observations. After training, intentions can be projected back through unrolled spike trains of the forward model essentially routing the intention-driven motor gradients towards the respective joints, which unfolds goal-direction navigation. We demonstrate that spiking neural networks can thus effectively control trunk-like robotic arms with up to 75 articulated degrees of freedom with near millimeter accuracy.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
